components/alertdialog/package.json

```diff
@@ -64,5 +64,6 @@
 	],
 	"publishConfig": {
 		"access": "public"
-	}
+	},
+	"componentGuidelinesName": "alert-dialog"
```

Comment:
This link is actually broken on the docs site. It doesn't use the hyphen. There are a few component that are in similar boats, where our docs site link is broken, but the intended page does exist. Would this PR be a place to fix that? That feels like a really low priority, especially if the docs site is going away anyways.